### PR TITLE
Rewrite VMWare Workstation virtualizing guide

### PR DESCRIPTION
**Pull request by Damillora**
I rewrote the guide still following the style of the VirtualBox guide and also the rewritten Parallels Desktop guide too.

The Haiku installation instructions are simple because they are already covered in the Installation Guide linked on the previous section.

Also noticing a `vmware-addons` package, used it, and added the information in the Additional steps section.sed it, and added the information in the Additional steps section.

Made as part of Google Code In 2017.

### CODE REVIEW

File: content/guides/virtualizing/vmware-workstation.md [hunk 1]
```diff
@@ -1,131 +1,111 @@
...
 
-##### Go to section:
+This How-To guide will describe the process of running Haiku on virtual machine (VM) using VMWare Workstation 14 and an Anyboot image file.
 
-*   [Running Haiku from a VM image.](#part_vmimage)
-*   [Installing and running Haiku from an ISO image.](#part_iso)
-*   [Additional Steps.](#part_additional)
-*   [Troubleshooting.](#part_trouble)
+The Anyboot image file can be obtained [here](http://localhost:8080/get-haiku). VMWare Player is available from VMWare for free [on their website](https://www.vmware.com/products/player/). There is also VMware Workstation, which is similar but has more features and is commercial. This means you have to pay for this version, but it does offer a 30 days trial. However you could just stick to the free one _(VMware Player)_ as you most likely will not need the extra features. You can both read about the commercial version's extra features and download it from its website [here](https://www.vmware.com/products/workstation/overview.html)
```
Comment (humdingerb) [inline content/guides/virtualizing/vmware-workstation.md]: http://localhost:8080/get-haiku   :)

File: test [hunk 2]
```diff
@@ -0,0 +1,2 @@
+asd
```
Comment (waddlesplash) [inline test]: this looks like a mistake. :P

File: content/guides/virtualizing/vmware-workstation.md (hunk 1 above)
Comment (Damillora) [reply content/guides/virtualizing/vmware-workstation.md]: Ah, brain burp. Fixed.

File: test (hunk 2 above)
Comment (Damillora) [reply test]: Ah, forgot to delete it while struggling with commit signing.

File: content/guides/virtualizing/vmware-workstation.md (hunk 1 above)
Comment (waddlesplash) [reply content/guides/virtualizing/vmware-workstation.md]: Doesn't seem to be fixed...?

Comment (Damillora) [reply content/guides/virtualizing/vmware-workstation.md]: Apparently I should not edit while on the way to school. Should be actually fixed now.

**Comment by waddlesplash** [comment]
Thanks!

**Pull request merged**